assets/credentials.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: openshift-gcp-pd-csi-driver-operator
+  namespace: openshift-cloud-credential-operator
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+spec:
+  serviceAccountNames:
+    - gcp-pd-csi-driver-controller-sa
+  secretRef:
+    name: gcp-pd-cloud-credentials
+    namespace: ${NAMESPACE}
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: GCPProviderSpec
+    predefinedRoles:
+      - roles/resourcemanager.tagUser
```

Comment:
This belongs to https://github.com/openshift/cluster-storage-operator/blob/master/manifests/03_credentials_request_gcp.yaml, where the rest of CSI driver permissions are.

Reply:
Yes, just change in cluster-storage-operator/manifests should be enough.

Reply:
suggestion incorporated.